packages/excalidraw/element/types.ts

```diff
@@ -137,7 +137,8 @@ export type ImageCrop = {
   y: number;
   width: number;
   height: number;
-  naturalDimension: [number, number];
+  naturalWidth: number;
+  naturalHeight: number;
```

Comment:
@ryan-di changed to individual props so that we don't have to modify undo/redo `change.ts` much to be able to handle nested objects.

Also it's more expressive/cleaner this way.